docs/core.adoc

```diff
@@ -31,13 +30,14 @@ Refer to the link:./getting-started[Getting Started] guide for more information
 
 == Usage
 
-The Core module supports parsing of the configuration file generated by the mobile CLI, in practice this feature is used internally by other modules of the SDK:
+The Core module supports parsing of the configuration file generated by the mobile CLI, in practice this feature is used internally by other modules of the SDK however:
 
 [source, javascript]
 ----
-import { MetricsService } from '@aerogear/core';
-import configuration from '../mobile-services.json';
+import { ConfigurationHelper, MetricsService } from '@aerogear/core';
+import appConfiguration from '../mobile-services.json';
 
-// utilizes Core to retrieve it's own configuration
-const metrics = new MetricsService(configuration);
+const helper = new ConfigurationHelper(appConfiguration);
+const metricsConfig = helper.getConfig(MetricsService.ID);
```

Comment:
I had replaced this with just instantiating the `MetricsService` due to https://github.com/aerogear/aerogear-js-sdk/pull/58#discussion_r183633054, because in practice the user would never need to do this.

I'm happy to keep it this way though, since it's pretty much the only thing that core does atm. If that's the case we can remove the `MetricsService` from the import at line 37

Reply:
Ah yes, good catch.